lib/modules/manager/gradle/parser.ts

```diff
@@ -937,19 +1031,59 @@ function tryMatch({
   return null;
 }
 
-export function parseGradle(
+async function parseInlineScriptFile(
+  scriptFile: string,
+  variables: PackageVariables,
+  recursionDepth: number,
+  packageFile = ''
+): Promise<SyntaxHandlerOutput> {
```

Comment:
```suggestion
): Promise<SyntaxHandlerOutput | null> {
```
or does the type already includes null?

Reply:
Already includes null: https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/gradle/types.ts#L50